server/pbenchinacan/container-build.sh

```diff
@@ -79,5 +81,31 @@ buildah run $container systemctl enable nginx
 buildah run $container systemctl enable rsyslog
 buildah run $container systemctl enable pbench-server
 
-# Create the container image.
-buildah commit $container localhost/pbench-server:${PB_SERVER_IMAGE_TAG}
+# Skip installing and configuring the Firewall
```

Comment:
Is there are particular reason why we need to note that we are skipping this step?  If so, it should probably be written here.  If not, perhaps we can remove this?